Kickstarter-iOS/Views/Cells/PledgeShippingLocationCell.swift

```diff
@@ -101,6 +101,13 @@ final class PledgeShippingLocationCell: UITableViewCell, ValueCell {
 
     self.amountLabel.rac.attributedText = self.viewModel.outputs.amountAttributedText
     self.shippingLocationButton.rac.title = self.viewModel.outputs.shippingLocationButtonTitle
+
+    self.viewModel.outputs.shippingLocationSelected
+      .observeForUI()
+      .observeValues { [weak self] shippingRule in
+        guard let self = self else { return }
+        self.delegate?.pledgeShippingCell(self, didSelectShippingRule: shippingRule)
```

Comment:
Seeing as `ShippingRule` is a type that explains this at the callsite, we can actually make this:

```swift
self.delegate?.pledgeShippingCell(self, didSelect: shippingRule)
```

Reply:
See later comment, I now realize exactly what the delegate is doing, it's really just presenting the rules, the actual selection will happen elsewhere.